modules/form/test/spec.coffee

```diff
@@ -0,0 +1,5 @@
+describe 'form', ->
+  it 'should have user facing text defined', ->
+    hx.userFacingText('form','missingRadioValue').should.equal('Please select one of these options')
+    hx.userFacingText('form','missingValue').should.equal('Please fill in this field')
+    hx.userFacingText('form','typeMismatch').should.equal('Please enter a valid value for this field')
```

Comment:
Added tests for the new user facing text - makes sure the dependency is added in the right place and that the text exists / hasn't changed